src/connectionHandler.ts

```diff
@@ -72,12 +72,19 @@ const initCmdObserver = async (
 
   const env = initEnvironment()
 
-  env.sendMessage.subscribe(msg => {
-    const embed = new MessageEmbed().setDescription(msg)
-    message.channel.send({
-      embeds: [embed]
-    })
-  })
+  env.sendMessage.subscribe(async msg => sendMessage({ msg, message })
```

Comment:
Kan använda async-await här om du föredrar det
```suggestion
  env.sendMessage.subscribe(async msg => {
    const sentMsg = await sendMessage({ msg, message })
    if (sentMsg.embeds[0].title?.startsWith('Queue')) {
      env.reprintQueueOnReaction.next(sentMsg)
    }
  })
```

Reply:
Hmm tror jag har kvar det som det är, men såg att jag missat asyncen på sent message i then så lägger till den